extension/src/counter_agg.rs

```diff
@@ -1,3 +1,5 @@
+pub mod gauge;
```

Comment:
Can you elaborate?

For the sake of keeping the change minimal, I've built this `gauge` mod on the existing `counter_agg` mod, so this needs to be contained within.

As a follow-up, I think we should factor the common bits out of `counter_agg` into a third mod (within the extension crate) both these modules would build on. We could make `gauge_agg` a sibling rather than child of `counter_agg` at that point.

Would that get us where you want to be?

Reply:
With the further separation of the aggregates we discussed last week, this became the obvious move after all. Done now. Thanks!